NDTensors/src/lib/GradedAxes/src/fusion.jl

```diff
@@ -6,6 +6,10 @@ OneToOne() = OneToOne{Bool}()
 Base.first(a::OneToOne) = one(eltype(a))
 Base.last(a::OneToOne) = one(eltype(a))
 
+gradedisequal(::AbstractUnitRange, ::OneToOne) = false
+gradedisequal(::OneToOne, ::AbstractUnitRange) = false
```

Comment:
Shouldn't these be true if the `AbstractUnitRange` runs from `1:1` and doesn't have any labels?

Reply:
Fixed in #1531 , remove from this PR.